airbyte-integrations/bases/connector-acceptance-test/connector_acceptance_test/tests/test_core.py

```diff
@@ -652,6 +656,11 @@ async def test_discover(self, connector_config, docker_runner: ConnectorRunner):
         assert len(catalog_messages) == 1, "Catalog message should be emitted exactly once"
         assert catalog_messages[0].catalog, "Message should have catalog"
         assert catalog_messages[0].catalog.streams, "Catalog should contain streams"
+        assert self.streams_are_uniquely_named(catalog_messages[0].catalog.streams), "Catalog should have uniquely named streams"
+
+    def streams_are_uniquely_named(self, streams) -> bool:
+        names = [stream.name for stream in streams]
+        return len(set(names)) == len(names)
```

Comment:
This is the change that I made, not sure why all the format tagged along

Reply:
@jbfbell how did you run format?

Reply:
i did not 🤷 maybe it was intelliJ

Reply:
looks like the Auto formatter got it back inline.

However thats probably not a good configuration we have going on that caused the intellij format